website/deploy/production/snack.env

```diff
@@ -1,6 +1,7 @@
 LEGACY_SERVER_URL=https://expo.io
 SERVER_URL=https://expo.dev
 API_SERVER_URL=https://exp.host
+SNACKPUB_URL=https://snackpub-3hn4fth2vq-uc.a.run.app
```

Comment:
These are stable urls?

Reply:
it seems so. between deployments and even delete/recreate the cloud run service, it still sticks on the same name. as the url is not visible to users, if it changed, we could still re-deploy snack with new urls. that's why i didn't put the cloud run service behind google cloud load balancer.

Reply:
It doesn't need to block this PR but let's set up a load balancer later